learningequality/kolibri-design-system
# Initial implementation of KImg component

## Description
This is the initial implementation of the KImg component, which will ultimately fulfill all the specifications of [the KImg epic](https://github.com/learningequality/kolibri-design-system/issues/368), but for this round only includes `src`, `altText`, `isDecorative`, `height`, `width`, `min-width`, `max-width`, `min-height`, and `max-height`.

#### Issue addressed

Addresses #369 


## Steps to test

1. `yarn dev`
2. `KImg` doc page should be accessible at http://localhost:4000/kimg
3. can edit any of the examples in docs/pages/kimg.vue to test functionality and see results reflected on the KImg doc page


## Testing checklist



## Reviewer guidance



## Comments
